tool/plugin/lib/plugin.dart

```diff
@@ -490,7 +490,7 @@ class BuildCommand extends ProductCommand {
         separator('Release mode (--release) implies --unpack');
       }
       if (!await performReleaseChecks(this)) {
-        return 1;
+        //return 1;
```

Comment:
This should still return `1` (to fail)?

Reply:
Good catch. Too much git commanding. Fixed.